### DISCUSSION TITLE
Change hljs theme

### DISCUSSION THREAD

**Discussion by pepone** [dotnet/docfx#8729] [Q&A]
Is there a way to change the hljs theme?

**Comment by KalleOlaviNiemitalo** [comment]
Perhaps <https://github.com/dotnet/docfx/pull/8682> will help with that.  This change is not yet in v2.66.2.

**Reply by pepone** [reply]
I saw that earlier but I cannot figure out how to do it

**Comment by pepone** [comment]
Seems the hljs style is hardcoded in 

https://github.com/dotnet/docfx/blob/fcf3d674ee8fad4c2b3c490e83f16151becae28e/templates/default/src/docfx.vendor.js#L5

I don't really see how we can switch, am I missing something?

**Reply by yufeih** [reply]
Correct, this is an area not customizable yet.

**Reply by ghost** [reply]
@yufeih Is this something that is on the near-term roadmap?  I appreciate it is a matter of personal taste, but C# example code would look much better in a theme like 'vs2015'.

If you could provide some bullet point guidance, maybe I could implement as  PR?

**Reply by yufeih** [reply]
I agree that it looks more familiar to C# devs. It appears we could use `vs` for the light theme and `vs2015` for the dark theme? If so a PR is welcome. The relevant code are: https://github.com/dotnet/docfx/blob/main/templates/modern/src/highlight.scss and https://github.com/dotnet/docfx/blob/main/templates/modern/src/docfx.scss#L14-L18

**Reply by wilkinson-steve** [reply]
Very happy to do a PR @yufeih and agree that `vs` and `vs2015` are an improvement for C# users.  I am just slightly worried that this may cause unexpected changes for non-C# users of docfx.  What do you think?  Is some sort of poll in order?

(On the basis that any change we make now is effectively hard-coded.)

**Reply by ghost** [reply]
See PR https://github.com/dotnet/docfx/pull/8761.

**Reply by yufeih** [reply]
@pepone What hljs styles would you like to use?

**Reply by pepone** [reply]
The proposed themes (vs/vs2015) seem good to me.
